README.md

```
@@ -204,15 +203,15 @@ Playwright 공식 AI 테스트 에이전트(planner/generator/healer)를 래핑
 | Skill | Purpose |
 |-------|---------|
 | `research-orchestrator` | 메인 진입점. 쿼리 → mode 감지 (quick/deep) → fan-out → synthesis-scout 합성. |
-| `exa-web-search` | web-scout 의 exa MCP + 4-tier fetch (exa → firecrawl → insane-search) 사용 가이드. |
+| `exa-web-search` | web-scout 의 exa MCP + 4-tier fetch (exa → brightdata → insane-search) 사용 가이드. |
 | `resource-finder` | github/hf-scout 의 검색 hygiene cheat-sheet. |
 
 **Agent team (6, all `opus`):**
 | Agent | Axis |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)